ember_debug/vendor/startup-wrapper.js

```diff
@@ -32,17 +32,6 @@ var EMBER_VERSIONS_SUPPORTED = {{EMBER_VERSIONS_SUPPORTED}};
       return;
     }
 
-    // If Ember doesn't exist, we should stop here to avoid issues with accessing `Ember.VERSION`
-    if (!Ember) {
-      return;
-    }
-
-    if (!versionTest(Ember.VERSION, EMBER_VERSIONS_SUPPORTED)) {
```

Comment:
this should still be done, so that inspector can load compatible versions for very old emberjs

Reply:
Hey @patricklx, please don’t pay too much attention to our PRs with a Draft status 🙏 Early feedback is often a good thing to have, but in the present case we have been doing a lot of experiments to see how to plug the inspector side and the Embroider side, and Chris has been cutting things that wouldn't exist in a Vite-only world just so that we can quickly use this branch against the Embroider side. I'll remove the draft status when it's clean and ready to review.